### ISSUE TITLE
test to be sure symbols insert correctly

### ISSUE THREAD

**Issue by mbjones** [NCEAS/metacatui#446]
I’ve been meaning to mention — when you develop the testing framework for MetacatUI, one important class of tests involves handling cut and paste from multiple character encodings and symbols that researchers use in their text; these include symbols like “smart quotes”, emdash, endash, degrees, angstroms, microns, macrons, accents, unicode characters, greek letters, math symbols like integrals and summation symbols, chinese and other ideographic characters, and many others. The current editor garbles much of that, and the data curation team fixes it manually. So can you be sure that we are executing good tests that ensure that pasted blocks of text with symbols from MS Word and other apps works properly and ends up encoded correctly, and displays in all of our software correctly?

**Comment by laurenwalker** [comment]
I did some manual testing with symbols in the Editor and saw no issues. It would be a good idea to make this part of the MetacatUI testing framework, though.

**Comment by csjx** [comment]
Yes, good point. I'll look into doing this with a straight unit test, but if we have to use Selenium or the like, we can look into that as well.
